<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>An update for YUView is available. Do you want to download and install the update now?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="checkUpdatesGroupBox">
     <property name="toolTip">
      <string>Should updates be installed automatically?</string>
     </property>
     <property name="whatsThis">
      <string>Should updates be installed automatically?</string>
     </property>
     <property name="title">
      <string>Check for updates</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>Should updates be installed automatically?</string>
        </property>
        <property name="whatsThis">
         <string>Should updates be installed automatically?</string>
        </property>
        <property name="text">
         <string>Update Setting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="updateSettingComboBox">
        <property name="toolTip">
         <string>Should updates be installed automatically?</string>
        </property>
        <property name="whatsThis">
         <string>Should updates be installed automatically?</string>
        </property>
        <item>
         <property name="text">
          <string>Automatically download and update</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always ask before an update</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
